build_conan.py

```diff
@@ -0,0 +1,81 @@
+import os
+import sys
+import subprocess
+
+
+if __name__ == '__main__' :
+
+  # Defaulted env var inputs - can override if necessary
+  CONAN = os.environ.get('CONAN', 'conan')
+  CONAN_HOME = os.environ.get('CONAN_HOME', f'{os.getcwd()}/.conan2')
+  os.environ['CONAN_HOME'] = CONAN_HOME # export it
+  OUTPUT_DIR = os.environ.get('OUTPUT_DIR', f'{os.getcwd()}/build')
+  CONAN_BUILD_PROFILE = os.environ.get('CONAN_BUILD_PROFILE', 'default')
+  CONAN_HOST_PROFILE = os.environ.get('CONAN_HOST_PROFILE', 'default')
+  SLM_BUILD_DYNAMIC_LIB = os.environ.get('SLM_BUILD_DYNAMIC_LIB', 'False').capitalize()
+
+  def no_conan_profile () :
```

Comment:
Why not use:

```
(venv) ➜  lbstanza-z3 git:(master) conan profile show -pr asdf    
ERROR: Profile not found: asdf
```

Here instead of digging around in `conan`'s internals?

Reply:
Good catch